src/common/components/CtaCard/CtaCard.theme.ts

```diff
@@ -0,0 +1,29 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type CtaCardVariants = VariantProps<typeof ctaCardTheme>;
+
+export const ctaCardTheme = tv(
+  {
+    slots: {
+      button: [
+        "flex h-fit w-full items-center justify-between self-stretch border p-6",
+      ],
+      ctaWrapper: ["flex items-center gap-3"],
+      icon: ["h-6 w-6"],
+      cta: ["text-lg font-semibold"],
+    },
+    variants: {
+      variant: {
+        primary: {
+          button: [

```

Comment:
hovered bg color of `primary` variant of `CtaCard` looks off to me (probably because the parent button component has "ghost" variant and applies that variant's hover bg.

From a UI perspective i think we can align "primary" `CtaCard` with the hover behaviour of the "like" button in `ReviewHeader` since the bg is the same (looks like its from button "secondary")

hovered:
![CleanShot 2024-05-06 at 15 51 00@2x](https://github.com/AfterClass-io/afterclass.io-v2/assets/68373112/adee306f-f78c-41c2-b174-4e4abd82b5aa)
![CleanShot 2024-05-06 at 15 54 24@2x](https://github.com/AfterClass-io/afterclass.io-v2/assets/68373112/9d29e8ea-9b88-4a00-8a90-3af23cada23e)

Reply:
good point! updated at 76f300fa690d02cdacb4a324ecf6f43006d2c53b & [67a1922](https://github.com/AfterClass-io/afterclass.io-v2/pull/111/commits/67a19220fa5d1e38ffe81833b41480f8f44facd9)

`<CtaCard/>` now takes button variants as props and pass it directly to the button

Review CTA uses secondary and Contribute CTA uses tertiary buttons